tools/CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# Azure SDK Tools Contribution Guidelines
+
+This is the repository of tool libraries used by Azure SDK purpose.
+For every newly developing tool, create a new folder with name of brief description of the tool. 
+Every tool should serve for single purpose.
+Do not add third party tools directly in your folder, use it as dependencies.
+
```

Comment:
tools should have entries inside the codeowners file.

Reply:
Add code owner section